test/integration/targets/docker_swarm_info/tasks/test_swarm_info.yml

```
@@ -125,6 +125,26 @@
          - 'output.docker_swarm_active == true'
          - 'output.docker_swarm_manager == true'
 
+  - name: Update swarm cluster to be locked
+    docker_swarm:
+      state: present
+      autolock_managers: true
+    register: autolock_managers_update_output
+
+  - name: Try to get docker_swarm_info and swarm_unlock_key
+    docker_swarm_info:
+      unlock_key: yes
+    register: output
+
+  - name: assert reading swarm facts and swarm unlock key
+    assert:
+      that:
+        - 'output.swarm_unlock_key'
```

I would make this a little bit more explicit:
```suggestion
        - 'output.swarm_unlock_key is string'
```